app/controllers/categoryController.js

```diff
@@ -0,0 +1,57 @@
+import CategoryModel from '../models/categoryModel';
```

Comment:
please insert some comments at the top of this file that explain what is contained in the file